src/stores/messages/MessagesDataProviderStore.ts

```diff
@@ -227,12 +249,21 @@ export default class MessagesDataProviderStore {
 	};
 
 	@action
-	public startNextMessagesChannel = (query: MessagesSSEParams) => {
+	public startNextMessagesChannel = (query: MessagesSSEParams, interval?: number) => {
 		this.searchChannelNext = new SSEChannel(
 			this.messagesStore.filterStore.messsagesSSEConfig.type,
 			query,
 			this.onNextChannelResponse,
 			this.onLoadingError,
+			interval
+				? e => {
```

Comment:
There could be running 2 parallel channels if softFilterIs applied. I think we can add keep_alive handlers there too.
For example if you stop prevChannel, prevChannel with filters could be running.
I thought maybe we can add some checks in keep_alive where we can check if channel without filters is running.
if it isn't we can check keep_alive timestamp with last fetched message. if its later or sooner depending on search direction we might want to stop it